ptrail/regularization/goemetric_curvature_and_turning.py

```diff
@@ -0,0 +1,82 @@
+"""
+    | Author: Abhishek Gujjar
+"""
+
+import numpy as np
+from typing import List, Union
+
+
+def traj_angles_wrapper(trj) -> Union[float, np.nan]:
+    """
+    Wrapper function to calculate trajectory angles using TrajAngles function.
+
+    Args:
+        trj: Input trajectory
+
+    Returns:
+        Trajectory angles or NaN if an exception occurs
+    """
+    try:
+        # Calculate trajectory angles using TrajAngles function
+        return TrajAngles(trj)
```

Comment:
Reach out to Yashar about the definitions of these functions, we need working code, can't keep it like this or merge it like this.